packages/browser/src/redux/BacktraceReduxMiddleware.ts

```diff
@@ -0,0 +1,32 @@
+import { Middleware, Action } from 'redux';
+import { BacktraceClient } from '../BacktraceClient';
+
+/**
+ *
+ * @param client BacktraceClient used to send breadcrumbs
+ * @param interceptAction A function that can be used to skip an action or filter out information from a dispatched action, such as PII, that shouldn't be sent to Backtrace. Return undefined to skip an action, otherwise whatever (potentially modified) action returned will be sent to Backtrace.
+ */
+export const createBacktraceReduxMiddleware = (
+    client: BacktraceClient,
+    interceptAction: (action: Action) => Action | undefined = (action) => action,
```

Comment:
Can we make this optional?
```suggestion
    interceptAction?: (action: Action) => Action | undefined = (action) => action,
```

Reply:
Since it has a default, isn't it already optional?

Reply:
oh, right, didn't see that one